docs/source/markdown/podman-network-reload.1.md

```diff
@@ -9,7 +9,7 @@ podman\-network\-reload - Reload network configuration for containers
 ## DESCRIPTION
 Reload one or more container network configurations.
 
-Rootful Podman relies on iptables rules in order to provide network connectivity. If the iptables rules are deleted,
+Rootfull Podman relies on iptables rules in order to provide network connectivity. If the iptables rules are deleted,
```

Comment:
Both spellings are present, it's really confusing.

Reply:
The preferred way should be 'rootful', one el, because that is the normative English suffix.